packages/fxa-payments-server/src/components/LoadingOverlay.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { selectors } from '../store';
+import { State } from "../store/types";
+
+import './LoadingOverlay.scss';
```

Comment:
This isn't final UX by any means, but we can tinker further with this global loader in #718